docs/www/configuration.md

```diff
@@ -465,6 +465,41 @@ pandaproxy_client:
   # Default: ""
   scram_password: ""
 
+# The Schema Registry provides a RESTful interface for Schema storage, retrieval, and compatibility.
+# To disable the Schema Registry, remove this top-level config node
+schema_registry:
+  # A list of address and port to listen for Schema Registry API requests.
+  # Default: 0.0.0.0:8082
+  schema_registry_api: 
+  - address: "0.0.0.0"
```

Comment:
I think we'd like to move to combining address and port into one line. Why separate them?

Reply:
Also this could be a one liner with `internal_address: "0.0.0.0:8081"` and `external_address: "0.0.0.0:18081"`.

Reply:
In fact since there seems to only be internal and external, it may also make sense to combine the below TLS block into this block.

Reply:
I don't necessarily disagree, but the precedent is set in the deserialisation code of redpanda: https://github.com/vectorizedio/redpanda/blob/dev/src/v/json/json.cc#L75

The number of listeners and their names are arbitrary,

Reply:
It looks like that is reading json, not yaml, so something is passing this yaml as json to redpanda? If so, that shim can be updated to take in a new format and spit out the old. This isn't critical right now, but it is something I'd like to address as an overall setup simplification / config simplicity perspective sooner than later, while we're still in some sort of period where we can make breaking changes (while supporting the old) without having a huge ecosystem broken.

Reply:
I've made suggestions for improved configuration layout in the past: https://github.com/vectorizedio/redpanda/issues/532, it seemed less than useful to make a breaking change until this is reviewed/merged: https://github.com/vectorizedio/redpanda/pull/553

Reply:
YAML conversion: https://github.com/vectorizedio/redpanda/blob/dev/src/v/config/configuration.h#L509